openmc/source.py

```diff
@@ -298,6 +298,10 @@ def __init__(self, r=(0., 0., 0.), u=(0., 0., 1.), E=1.0e6, time=0.0, wgt=1.0,
         self.surf_id = surf_id
         self.particle = particle
 
+    def __repr__(self):
+        name = self.particle.name.lower()
+        return f'<SourceParticle: {name} at E={self.E:.3e} eV>'
```

Comment:
```suggestion
        return f'<SourceParticle: {name} at E={self.E} eV>'
```
Just wondering if we want to return the energy without formatting to 3 significant figures?